src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-list/cephfs-list.component.ts

```diff
@@ -42,6 +78,39 @@ export class CephfsListComponent extends ListWithDetails implements OnInit {
         cellTransformation: CellTemplate.checkIcon
       }
     ];
+    this.tableActions = [
+      {
+        name: this.actionLabels.CREATE,
+        permission: 'create',
+        icon: Icons.add,
+        click: () => this.router.navigate([this.urlBuilder.getCreate()]),
+        canBePrimary: (selection: CdTableSelection) => !selection.hasSelection
+      },
+      {
+        name: this.actionLabels.EDIT,
+        permission: 'update',
+        icon: Icons.edit,
+        click: () => this.editAction()
+      },
+      {
+        permission: 'delete',
+        icon: Icons.destroy,
+        click: () => this.deleteVolumeModal(),
+        name: this.actionLabels.DELETE,
```

Comment:
Remove instead of Delete?
```suggestion
        name: this.actionLabels.REMOVE,
```